components/admin_console/workspace-optimization/dashboard.tsx

```diff
@@ -244,26 +246,70 @@ const WorkspaceOptimizationDashboard = (props: Props) => {
     }, []);
 
     const accData: AccordionItemType[] = Object.entries(data).map(([accordionKey, accordionData]) => {
-        const items = accordionData.items.map((item) => (
-            <AccordionItem
-                key={`${accordionKey}-item_${item.id}`}
-                iconColor={getItemColor(item.status)}
-            >
-                <h5><i className={classNames('icon', {'icon-check-circle-outline': item.status === 'none', 'icon-alert-outline': item.status === 'warning', 'icon-alert-circle-outline': item.status === 'error'})}/>{item.title}</h5>
-                <p>{item.description}</p>
-            </AccordionItem>
-        ));
+        const chipsInfo: { [key in ChipsInfoKey]: number } = {
+            info: 0,
+            warning: 0,
+            error: 0,
+            none: 0,
+        };
+        const items: React.ReactNode[] = [];
+        accordionData.items.forEach((item) => {
+            items.push((
+                <AccordionItem
+                    key={`${accordionKey}-item_${item.id}`}
+                    iconColor={getItemColor(item.status)}
+                >
+                    <h5><i className={classNames('icon', {'icon-check-circle-outline': item.status === 'none', 'icon-alert-outline': item.status === 'warning', 'icon-alert-circle-outline': item.status === 'error'})}/>{item.title}</h5>
+                    <p>{item.description}</p>
+                </AccordionItem>
+            ));
+            if (chipsInfo[item.status] !== undefined) {
+                chipsInfo[item.status] += 1;
+            }
+        });
+        const chipsList = Object.entries(chipsInfo).map(([chipKey, count]) => {
+            if (count === 0) {
```

Comment:
Sorry, I forgot to mention this on the last review:

We possibly need to check for `chipKey === 'none'` here as well or it might output chips that are producing errors.

Or maybe we remove the `'none'` from the union and have undefined instead so we can check for existence in the above `forEach`?

third option: rename `'none'` to `'ok'`?

WDYT @pablovelezvidal ?

Reply:
I think we should change it to `ok` or `success` that way we keep the four common states of the chip.

Reply:
could we then use `!item.status` as a flag to not render the item? WDYT?

Reply:
I think is better use the count of occurrences.

Reply:
see comment on line 256 ... I should have written that there from the beginning 🙇🏼